daemon/containerd/image_pull.go

```diff
@@ -118,7 +119,8 @@ func (i *ImageService) pullTag(ctx context.Context, ref reference.Named, platfor
 			sentSchema1Deprecation = true
 		}
 		if images.IsLayerType(desc.MediaType) {
-			progress.Update(out, desc.Digest.String(), "Pulling fs layer")
+			id := stringid.TruncateID(desc.Digest.String())
+			progress.Update(out, id, "Pulling fs layer")
 		}
```

Comment:
We really need to have a close look at these progress things. This should (really) be all part of the presentation, not the API.  I realise we can't easily change all of that right now, so for now, it's probably OK to align with the existing behavior, but it's really nasty.

Also see;

- https://github.com/moby/moby/issues/46528